tapchannel/aux_traffic_shaper.go

```diff
@@ -196,6 +189,14 @@ func (s *AuxTrafficShaper) PaymentBandwidth(htlcBlob,
 		// means in terms of satoshis. But the satoshi amount doesn't
 		// matter too much here, we just want to signal that this
 		// channel _does_ have available bandwidth.
+
+		// Check if the current link bandwidth can afford sending out the htlc
```

Comment:
nit: I think this block should probably be above the other comment in this block? Since that comment explicitly references the `btcutil.MaxSatoshi` that is returned.